packages/forma-36-react-components/src/components/Asset/README.mdx

```diff
@@ -0,0 +1,107 @@
+import { Asset } from './Asset';
+
+# Asset
+
+[Source code](https://github.com/contentful/forma-36/blob/master/packages/forma-36-react-components/src/components/Asset/Asset.tsx)
```

Comment:
I don't think we need this link in the README, do we?

I mean if the person is reading this, then they are in github already